From c31339e0de76884ab2233da68c04994c99e339e1 Mon Sep 17 00:00:00 2001
From: Weng Xuetian <wengxt@gmail.com>
Date: Tue, 26 Jan 2016 17:30:01 -0800
Subject: [PATCH] Fix lookupProperty for xcb backend

There's no point check return atom value. Original implementation tends
to check the pointer is null, instead of the atom value.
---
 src/common/x11inputdevice-xcb.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/x11inputdevice-xcb.cpp b/src/common/x11inputdevice-xcb.cpp
index f499b7d..4d70bc1 100644
--- a/src/common/x11inputdevice-xcb.cpp
+++ b/src/common/x11inputdevice-xcb.cpp
@@ -526,7 +526,7 @@ bool X11InputDevice::lookupProperty(const QString& property, X11InputDevice::Ato
 {
     Q_D(const X11InputDevice);
 
-    if (!isOpen() || property.isEmpty() || atom == XCB_ATOM_NONE) {
+    if (!isOpen() || property.isEmpty()) {
         return false;
     }
 
